web/uk-base-world/README.md

```diff
@@ -71,12 +82,12 @@ To run the script and bring up a local instance of the UK Base World visualisati
    - Add your API key to a file at `./visualisation/mapbox_api_key`
 4. Add the data files:
    - Add the processed DUKES 2023 CSV to the `./inputs/data/uk_base_world/dukes_2023` directory.
-   - Add the OntoEIP OWL and TTL files to the `./inputs/data/ontoeip` directory (note that these need to be in a flat structure, no subdirectories).
+   - Add the OntoEIP OWL and TTL files to the `./inputs/data/uk_base_world/ontoeip` directory (note that these need to be in a flat structure, no subdirectories).
```

Comment:
Appreciate that you've added READMEs within the specific data folders, but I doubt people will know to look for them. Please add an entry to this list for each new data set that needs copying in.